sdk/tests/client/message_interface.rs

```diff
@@ -51,7 +51,7 @@ async fn generate_addresses() {
 #[tokio::test]
 #[should_panic]
 async fn build_and_post_block() {
-    // This test uses dotenv, which is not safe for use in production
+    //This test uses secrets in environment variables for simplicity which should not be done in production.
```

Comment:
```suggestion
    // This test uses secrets in environment variables for simplicity which should not be done in production.
```

Reply:
sorry ... didn't see this in time so I cannot commit it anymore. will add it tho